### DISCUSSION TITLE
Loading Tiled Map

### DISCUSSION THREAD

**Discussion by mattoni** [StarArawn/bevy_ecs_tilemap#191] [Q&A]
Hello, thanks for the awesome plugin. I have a question about loading maps from Tiled. 

The readme refers to a `tiled_map` feature, but that feature doesn't seem to be present. The tiled examples rely on `tiled.rs` in the examples folder. If I want to load in a tiled map, do I need to copy/reimplement this file? Or is there a way to load the tiled map without needing to copy that file?

Thanks!

**Comment by StarArawn** [comment]
The README does need to be updated. The tiled feature was replaced with an example showing how to load a tiled map editor map file into bevy. One of the reasons behind this is that tiled provides a lot of additional data not related to the tile map. It makes more sense to have a separate crate to implement these features or have users implement it themselves. You should be able to see how to load a tiled map by running: `cargo run --example tiled`.
